src/Mooc/Courses/Application/Find/CoursesResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodelyTv\Mooc\Courses\Application\Find;
+
+use CodelyTv\Shared\Domain\Bus\Query\Response;
+
+final class CoursesResponse implements Response
+{
+    /** @var array */
+    private $courses;
+
+    /**
+     * CoursesResponse constructor
+     */
```

Comment:
remove all dockblocks in all classes
```suggestion
    /**
     * CoursesResponse constructor
     */
```